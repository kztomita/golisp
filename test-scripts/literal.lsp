(print 1)
(print 1.2)
(print "foo")
(print t)
(print nil)
