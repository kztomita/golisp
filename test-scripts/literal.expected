1
1.2
"foo"
t
nil
